modules/optimization/examples/bimaterial_mesh/gold/main_out_OptimizationReporter_0001.csv

```diff
@@ -0,0 +1,12 @@
+diffusivity_values,grad_diffusivity_values,measurement_time,measurement_values,measurement_xcoord,measurement_ycoord,measurement_zcoord,misfit_values,simulation_values
+5,0,0,1.5499999950466,0,-3.99,0,0,1.5499999957394
+10,0,0,122.15999960961,0,-3.192,0,0,122.15999966422
+0,0,0,230.74999926258,0,-2.394,0,0,230.74999936573
+0,0,0,326.45999895672,0,-1.596,0,0,326.45999910265
+0,0,0,409.18999869233,0,-0.798,0,0,409.18999887525
+0,0,0,479.99999846604,0,0,0,0,479.99999868061
+0,0,0,508.43499922095,0,0.798,0,0,508.43499931579
+0,0,0,530.90999981763,0,1.596,0,0,530.90999981784
+0,0,0,546.895000242,0,2.394,0,0,546.89500017492
+0,0,0,556.44000049541,0,3.192,0,0,556.44000038813
+0,0,0,559.97500058926,0,3.99,0,0,559.9750004671
```

Comment:
One minor comment: since the new example is solving the same bi-material problem using a new implementation, I suppose it would be better to use the same gold files between the two (i.e., keep only one gold file and create a symbolic link for the other test) - I think they already match when the TAO solve finishes, the difference should be from the initial condition.

Reply:
I couldn't do this because the name of one of the columns is different between when the parameter is defined on a mesh